generators/client/templates/angular/src/main/webapp/env.js.ejs

```diff
@@ -0,0 +1,2 @@
+window.__env = window.__env || {};
+window.__env.myDynamicVariable = 'http://127.0.0.1:8090';
```

Comment:
Can you use a more meaningful name ? What is this variable used for. If it is not used and just an example then it should be commented out.

Reply:
the `window.__env` varaible is just a way to group all variable in one place to avoid polluting the window. `myDynamicVariable` is just an example one how to use the `__env` object.

It was not commented for the reasons explained in my previous comment. but I can definitely add some comments, what do you think about this changes.

Reply:
Great.